lib/tasks/configTenantAdmin.js

```diff
@@ -100,6 +100,15 @@ function questions(done) {
                return !values.email && !Options.email;
             },
          },
+         {
+            name: "url",
+            type: "input",
+            message: "Enter the Tenant Administrator URL:",
+            default: `http://localhost:80`,
```

Comment:
let's make this default more descriptive for the live tenant:  `https://[tenantKey].site.url`